_posts/2022-11-10-october-rubygems-updates.md

```diff
@@ -0,0 +1,34 @@
+---
+title: October 2022 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+author_email: laurandidi21@gmail.com
+---
+
+Welcome to the RubyGems monthly update! As part of our efforts at Ruby Central, we publish a recap of the work that we’ve done the previous month. Read on to find out what updates were made to RubyGems and RubyGems.org in October.
+
+## RubyGems News
+This month in RubyGems, we released new versions of RubyGems [3.3.23](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3323--2022-10-05), [3.3.24](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3324--2022-10-17) and Bundler [2.3.23](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2323-october-5-2022), [2.3.24](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2324-october-17-2022).
+
+
+The following improvements and fixes are also included in these releases (see the changelog for more information):
+
+- made the `util/rubocop` script use `in-tree` Bundler - [#5979](https://github.com/rubygems/rubygems/pull/5979).
+- added extra resolver spec groups for Ruby platform when required - [#5698](https://github.com/rubygems/rubygems/pull/5698).
+- added `SHA256` in test certificates - [#5982](https://github.com/rubygems/rubygems/pull/5982).
+- GitHub DSL documentation was updated to reflect that it uses the HTTPS protocol under the hood instead of the git protocol. - [#5993](https://github.com/rubygems/rubygems/pull/5993).
+- allow JRuby to pass keywords to `Kernel#warn` - [#6002](https://github.com/rubygems/rubygems/pull/6002).
+- specified all GitHub sources use the HTTPS secure protocol by default - [#6026](https://github.com/rubygems/rubygems/pull/6026).
+
```

Comment:
I would also add the following bullet points:

* Several issues with `Gem::Platform` handling were fixed in `musl` platforms ([#5915](https://github.com/rubygems/rubygems/pull/5915)), in `arm` platforms with `wabi` modifiers ([#5957](https://github.com/rubygems/rubygems/pull/5957)), and to properly deal with string parameters when comparing ([#5939](https://github.com/rubygems/rubygems/pull/5939)).
* We improved handling of permanent redirect responses when pushing gems. - [#5931](https://github.com/rubygems/rubygems/pull/5931).
* We fixed an obscure issue affecting file extraction of some specific `.gem` packages. - [5906](https://github.com/rubygems/rubygems/pull/5906/files)
* We migrated the GitLab CI template generated by `bundle gem` to be the one now recommended by GitLab.